source/reference/beta-program.txt

```diff
@@ -97,6 +97,11 @@ Beta Features
        guidelines and restrictions for migrating between older server 
        versions, you must have access to the {+c2c-beta-program-short+} binary.
 
+.. note:: 
+
+   Mongosync :ref:`c2c-api-reverse` mode is not compatible with any beta 
+   features. 
```

Comment:
Apologies for the confusion here, could we remove this note from the Reverse page? It would be preferable to keep beta-related docs scoped to the beta page except for cases when we actively want to call attention to beta features. In this case, since Reverse isn't compatible, I'd recommend we didn't mention the beta.